### PR TITLE
Pre loaders

### PR DESCRIPTION
**Pull request by himanish-star**
@shikhar-scs, Please review

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -6,4 +6,3 @@ python_version = "2.7"
 "textblob" = "==0.15.1"
 "jsonschema" = "==2.6.0"
 "pandas" = "==0.21.0"
-w
```
Comment (himanish-star) [inline Pipfile]: Does this have some significance? @PikkaPikkachu

File: semaphores [hunk 2]
```diff
@@ -1,52 +0,0 @@
- We looked at using locks to provide mutual exclusion
- Locks work, but they have some drawbacks when
-critical sections are long
-◆ Spinlocks – inefficient
```
Comment (himanish-star) [inline semaphores]: @shikhar-scs, kya hai ye?

File: Pipfile (hunk 1 above)
Comment (PikkaPikkachu) [reply Pipfile]: Yes

**Comment by himanish-star** [comment]
All right merging then :+1:

**Pull request merged**